src/main/resources/api/swagger.yaml

```diff
@@ -12,104 +12,103 @@ produces:
 paths:
   /dependencies:
     get:
-      summary: Topologicaly sorted list of features.
```

Comment:
Since the description is also just a one-liner, there is no reason for keeping a summary. It is just redundant information. I would add a summary when the actual description is more extensive.